resources/zib/zib-MedicationAdministration2.xml

```diff
@@ -0,0 +1,318 @@
+    <element id="MedicationAdministration.supportingInformation">
+      <path value="MedicationAdministration.supportingInformation" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="type" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="MedicationAdministration.supportingInformation:administrationAgreement">
+      <path value="MedicationAdministration.supportingInformation" />
+      <sliceName value="administrationAgreement" />
+      <short value="AdministrationAgreement" />
+      <definition value="Administration agreement on which this administration is based." />
+      <alias value="Toedieningsafspraak" />
+      <max value="1" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-AdministrationAgreement" />
+      </type>
+      <mapping>
+        <identity value="MedicationAdministration2-v1.1.1-2020EN" />
+        <map value="NL-CM:9.13.23237" />
+        <comment value="AdministrationAgreement" />
+      </mapping>
+    </element>
+    <element id="MedicationAdministration.supportingInformation:administrationAgreement.type">
+      <path value="MedicationAdministration.supportingInformation.type" />
+      <min value="1" />
+      <fixedUri value="MedicationDispense" />
+    </element>
+    <element id="MedicationAdministration.effective[x]">
+      <path value="MedicationAdministration.effective[x]" />
+      <slicing>
+        <discriminator>
+          <type value="type" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="MedicationAdministration.effective[x]:effectiveDateTime">
+      <path value="MedicationAdministration.effective[x]" />
+      <sliceName value="effectiveDateTime" />
+      <short value="AdministrationDateTime" />
+      <definition value="The date and time at which the medication was administered." />
+      <alias value="ToedieningsDatumTijd" />
+      <type>
+        <code value="dateTime" />
+      </type>
+      <mapping>
+        <identity value="MedicationAdministration2-v1.1.1-2020EN" />
+        <map value="NL-CM:9.13.21193" />
+        <comment value="AdministrationDateTime" />
+      </mapping>
+    </element>
+    <element id="MedicationAdministration.performer">
+      <path value="MedicationAdministration.performer" />
+      <short value="Administrator" />
+      <definition value="The concept describes the person who administered the product. This is a professional authorised administrator, the patient themselves or the caregiver, for example." />
+      <alias value="Toediener" />
+      <max value="1" />
+      <mapping>
+        <identity value="MedicationAdministration2-v1.1.1-2020EN" />
+        <map value="NL-CM:9.13.21196" />
+        <comment value="Administrator" />
+      </mapping>
+    </element>

```

Comment:
backbone issue missing in the known issue .yml

Reply:
Will be added